packages/components/pill/src/Pill.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { cx } from 'emotion';
+import { CommonProps, Box } from '@contentful/f36-core';
+import { TabFocusTrap } from '@contentful/f36-utils';
+import { Drag, Close } from '@contentful/f36-icons';
+import { PillVariants } from './types';
+import getPillStyles from './styles';
+
+export interface PillProps
+  extends React.HTMLAttributes<HTMLElement>,
+    CommonProps {
+  label: string;
```

Comment:
Ideally, we would have jsdoc comments for all 5 properties here, because there are picked up by tsdoc generator and end up on the documentation page.